src/main.js

```diff
@@ -285,23 +297,29 @@ function handleClose(streams, children, childrenInfo) {
         if (aliveChildren.length === 0) {
             exit(exitCodes);
         }
-
-        if (config.killOthers && !othersKilled) {
-            othersKilled = true
+        if (!othersKilled) {
+          othersKilled = true;
```

Comment:
Hmmm... Could you elaborate? I don't think I follow. One of the tests demonstrates that this is not the case (unless I'm mistaken). What _should_ the code look like?